domain/src/main/java/me/androidbox/domain/util.kt

```diff
@@ -0,0 +1,14 @@
+package me.androidbox.domain.agenda.usecase
+
+import java.util.Locale
+
+fun String.toInitials(): String {
+    val listOfName = this.trim().split(" ")
+    var initials = ""
+
+    listOfName.forEach { name ->
+        initials += name.first()
+    }
+
+    return initials.uppercase(Locale.getDefault())
+}
```

Comment:
Some questions:
1. What if the full name consists of first, middle and last name? I think the initials should then be from first and last name ignoring the middle name.
2. What if the user only provides their first name as the full name? Then I think it should show the first 2 chars
3. What if there are 2 spaces in the name? Not 100% what `split` would do then but it might result in an empty string and crash here

Reply:
@philipplackner I will write some test cases for those scenarios